From: Philip Withnall <withnall@endlessm.com>
Date: Wed, 29 Aug 2018 12:12:05 +0100
Subject: glib: Remove remaining references to __int64

This is a follow-up to 7e821441c482917e54435a07893272d87d3ad9e5 and
e154e3325eb7274b8164f8d7a5e0f335646c2bb7 removing some remaining
references to __int64 which are no longer necessary.

Signed-off-by: Philip Withnall <withnall@endlessm.com>

Bug: https://gitlab.gnome.org/GNOME/glib/issues/1313
Origin: upstream, 2.59.0, commit:eeadd57a1428ef4617fc4c5f401f744e541478d8
---
 glib/gscanner.c | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/glib/gscanner.c b/glib/gscanner.c
index 389376c..9acf3df 100644
--- a/glib/gscanner.c
+++ b/glib/gscanner.c
@@ -1697,12 +1697,7 @@ g_scanner_get_token_i (GScanner	*scanner,
       *token_p = G_TOKEN_FLOAT;
       if (scanner->config->store_int64)
         {
-#ifdef _MSC_VER
-          /* work around error C2520, see gvaluetransform.c */
-          value_p->v_float = (__int64)value_p->v_int64;
-#else
           value_p->v_float = value_p->v_int64;
-#endif
         }
       else
 	value_p->v_float = value_p->v_int;
